doc/source/changes/version_0_24.rst.inc

```diff
@@ -18,6 +18,8 @@
      a0   0   1
      a1   2   3
 
+* added new boolean argument 'overwrite' to Session.save method. If True, overwrite existing file if any.
```

Comment:
* we should also mention other "save" methods: to_excel, etc. which are also impacted by this change.
* I don't think it is clear enough that the default behaviour changed from overwrite=False to overwrite=True